src/hack/README.rst

```diff
@@ -0,0 +1,30 @@
+=========================================
+Microsoft Azure CLI 'hack' Command Module
+=========================================
+
+A bootstrapping application for quickly creating `web space <https://azure.microsoft.com/en-us/services/app-service/>`_, a database (Cosmos DB, SQL Server, or MySQL), and a `Cognitive Services <https://azure.microsoft.com/en-us/services/cognitive-services/>`_ key.
+
+Creates an Resource Group, an App Service Plan and App Service, database server and database, and optionally enables Cognitive Services. All items are placed in a single resource group for easy cleanup post hack. Uses **Standard** pricing.
+
+-----
+Usage
+-----
+
+::
+    az hack up -n demoname -d MySQL -r python -ai
+
+--------
+Switches
+--------
+
+**--name -n**
+Name of the application. This will be used to name the Resource Group, App Service Plan, App Service, database server and database. Name must contain only letters and numbers, and start with a letter.
```

Comment:
I have one suggestion here, while resource group must be unique in a subscription, database name for instance must be globally unique, so probably when creating database it would be good to add random prefix.
but this is something that could be done later.

Reply:
Thanks for the suggestion! I'm going to update the code to have the user provide the first 10 characters as a base, and then i'll append 6 random (or something like that)